### PR TITLE
Update fovt.md

### PR DESCRIPTION
**Pull request by megbalk**
Ramona is no longer the contact, I am the one that maintains this ontology. Thanks!

### CODE REVIEW

**Comment by cthoyt** [comment]
@megbalk would you please update the label and github to also point to you?

**Comment by megbalk** [comment]
@cthoyt which label and github url?

**Comment by megbalk** [comment]
done!

File: ontology/fovt.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ build:
   system: git
   path: "."
 contact:
-  email: rlwalls2008@gmail.com
-  label: FuTRES Ontology of Vertebrate Traits
-  github: https://github.com/futres/fovt
+  email: meghan.balk@gmail.com
+  label: Meghan Balk
+  github: https://github.com/megbalk
```
Comment (cthoyt) [inline ontology/fovt.md:15]:
```suggestion
  github: megbalk
```

**Review by cthoyt — APPROVED** [review]

**Comment by matentzn** [comment]
No more changes so closing.

**Comment by cthoyt** [comment]
Unfortunately there was some miscommunication as this was supposed to be merged before #1725, which was rebased on top of this branch :(

**Comment by matentzn** [comment]
Changes came in so no problem :P Your concern for "contribution tracking" is as always very kind.

**Pull request closed without merging**